pkg/kcp/provider/azure/redisinstance/createPrivateDnsZoneGroup.go

```diff
@@ -0,0 +1,64 @@
+package redisinstance
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork/v5"
+	"github.com/kyma-project/cloud-manager/api/cloud-control/v1beta1"
+	"github.com/kyma-project/cloud-manager/pkg/composed"
+	azureutil "github.com/kyma-project/cloud-manager/pkg/kcp/provider/azure/util"
+	"github.com/kyma-project/cloud-manager/pkg/util"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+)
+
+func createPrivateDnsZoneGroup(ctx context.Context, st composed.State) (error, context.Context) {
+	state := st.(*State)
+	logger := composed.LoggerFromCtx(ctx)
+	if state.privateDnsZoneGroup != nil {
+		return nil, nil
+	}
+	if state.privateEndPoint == nil {
+		logger.Info("Can not create Azure Private DnsZone Group, Private EndPoint is not loaded, reque")
+		return composed.StopWithRequeueDelay(util.Timing.T1000ms()), ctx
+	}
+	logger.Info("Creating Azure Private DnsZone")
+	resourceGroupName := state.resourceGroupName
+	privateDnsZoneGroupName := state.ObjAsRedisInstance().Name
+	privateEndPointName := ptr.Deref(state.privateEndPoint.Name, "")
+	privateDnsZoneInstanceName := "privatelink.redis.cache.windows.net"
```

Comment:
You have this literal `privatelink.redis.cache.windows.net` defined in [pkg/kcp/provider/azure/util/resources.go](https://github.com/kyma-project/cloud-manager/pull/711/files#diff-4c82137b5462a6f8a47662a19c22d8cbdda05fb8e4380eaded9985f7367db9fa)

Should it be reused?

Reply:
nice catch, util func reused